docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -147,6 +147,8 @@ Elliptic Curve Key Exchange algorithm
         ... ).public_key()
         >>> shared_key = private_key.exchange(ec.ECDH(), peer_public_key)
 
+    When performing an ephemeral form of this exchange (commonly written ECDHE
+    or EECDH) you must generate a new private key for each ``exchange``.
```

Comment:
Can we use stronger prose here, like "Unless you are implementing a really bad protocol, you should be generating an ew private key for each exchange operation" (and link to the generate prive key docs), and maybe a link to a good reference on FS as well?